tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -1,98 +1,148 @@
 import { sortBy } from 'common/collections';
-import { useBackend } from "../backend";
+import { useBackend, useLocalState } from "../backend";
 import { Window } from "../layouts";
-import { NanoMap, Box, Table, Button } from "../components";
+import { NanoMap, Box, Table, Button, Tabs, Icon, NumberInput } from "../components";
 import { TableCell } from '../components/Table';
+import { COLORS } from '../constants.js';
 
 export const CrewMonitor = (props, context) => {
   const { act, data } = useBackend(context);
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 0);
   const crew = sortBy(
-    crewmember => crewmember.name,
+    cm => cm.name,
   )(data.crewmembers || []);
+  const [
+    mapZoom,
+    setZoom,
+  ] = useLocalState(context, 'number', 1);
+  let body;
+  // Data view
+  if (tabIndex === 0) {
+    body = (
+      <Table>
+        <Table.Row header>
+          <Table.Cell>
+            Name
+          </Table.Cell>
+          <Table.Cell>
+            Status
+          </Table.Cell>
+          <Table.Cell>
+            Location
+          </Table.Cell>
+        </Table.Row>
+        {crew.map(cm => (
+          <Table.Row key={cm.name}>
```

Comment:
For in-game readability, I think lines should be spaced by a small margin (`0.25rem` or `0.5rem`) and be highlighted on hover. You can use the `onMouseEnter`/`onMouseLeave` events OR make a dedicated `.scss` file (which I would prefer!)